_posts/2018-06-13-Exploring-The-vRo-Plugin-Part1.md

```diff
@@ -0,0 +1,374 @@
+
+![VRO Inventory Add Enterprise Start](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-Start.png)
+
+{:start="4"}
+1. A small pop-up window appears indicating that the workflow has begun exection  :
+ 
+![VRO Inventory Add Enterprise Running](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-Running.png)
+
+{:start="5"}
+1.  If the workflow in the previous step completed successfully, then the new enterprise will be visible as an object in the VRO Inventory.  Expand the `Me` object from the Inventory menu and then right-click the `Enterprises` object and select the `Reload` option from the small pop-up menu.  The most recent list of Enterprise objects from your instance of VSD will be displayed :
+
+![VRO Add Enterprise Reload VRO](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-Reload-VRO.png) ![VRO Add Enterprise See VRO](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-See-VRO.png)
+
+{:start="6"}
+1.  When a VSD object is selected from the VRO Inventory menu in the left pane, its attributes are displayed in the right pane.  For our purposes, we are interested in the ID attribute of the new enterprise object.  It may be necessary to scroll the screen in order to find the ID attribute.  Make note of the ID value as we will use it later as a validation item when we create custom workflows :
+
+![VRO Inventory Add Enterprise See ID Value](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-See-ID-Value.png)
+
+{:start="7"}
+1.  The new enterprise will also be visible as an object in `VSA` (Virtualized Services Architect), where its attributes can be displayed.  Select the new enterprise object and then choose the `Inspect` option from the small pop-up menu.  Ensure that the  Enterprise ID value matches the value from the previous step :
+
+![VRO Inventory Add Enterprise See VSA](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-See-VSA.png) ![VRO Inventory Add Enterprise See VSA ID](/img/posts/exploring-vro-part1/VRO-Inventory-Add-Enterprise-See-VSA-ID.png)
+
+# Building a Custom Workflow
+
+The Nuage Networks VSP vRO Plugin provides an extensive set of workflows that can perform basic maintenance tasks on VSD objects.  In addition, those workflows can be combined with your own custom designed workflows to perform more complex tasks in your network environment.
+
+In these next sections, we will design a custom workflow that invokes 3 Nuage VSP plugin workflows to find, edit and then delete the VSD Enterprise object that we created in the previous section.
+
+## <span style="color: purple">Create the Custom Workflow</span>

```

Comment:
Is there a reason why you are setting a specific color?

Reply:
I thought a different colour for the sub headings made the content more readable since there are so many sections.  Not a big deal to remove.  Do you prefer just back or a different one?